src/libraries/Microsoft.PowerFx.Core/IR/IRTranslator.cs

```diff
@@ -591,33 +590,40 @@ private static IntermediateNode BlankToEmptyString(IntermediateNode arg)
 
                 // We are not using coalesce, because coalesce doesn't considers empty string as blank.
                 return new UnaryOpNode(convertedIRContext, UnaryOpKind.BlankToEmptyString, arg);
-            }
-
-            /// <summary>
-            /// This is not a generic arg concatenate function, but a special case for the Concatenate function,
-            /// used for Binary Concatenate operator.
-            /// </summary>
-            private static IntermediateNode ConcatenateArgs(IntermediateNode arg1, IntermediateNode arg2, IRContext irContext)
+            }
+
+#pragma warning disable CS1570 // XML comment has badly formed XML

```

Comment:
>// XML comment has badly formed XML [](http://example.com/codeflow?start=31&length=35)

Can we make it well-formed, instead of disabling this warning? That will help with intellisense. I'm guessing that the `&` characters will have to be escaped as `&amp;`